kubernetes.js

```diff
@@ -329,12 +335,37 @@ const createService = async (project, options) => {
 const createIngress = async (project, options) => {
     const prefix = project.safeName.match(/^[0-9]/) ? 'srv-' : ''
 
+    const url = new URL(project.url)
```

Comment:
On line 244 you guard against `project.url` being undefined. Do we need a similar check here?

Reply:
No, because it is set and saved on line 320-321 so should be in place when the `createIngress()` is called on 375-377

Reply:
Thanks - missed that bit.